src/app/servicesPage/services.js

```diff
@@ -32,50 +34,128 @@ angular.module( 'vyomo.servicesPage', [
 
         function _sortAccordingtoPrice(packagesList) {
             return packagesList.sort(function (a, b) {
-                                a = parseInt(a.services[0].cost_including_discount, 10);
-                                b = parseInt(b.services[0].cost_including_discount, 10);
+                                a = parseInt(a.cost, 10);
+                                b = parseInt(b.cost, 10);
                                 return (a - b);
                             });
         }
 
 
+        function addToCart(){
+            if(this.quantity !== undefined){
+                this.quantity += 1;
+            }
+            cart.addItem(this);
+        }
+
+        function removeFromCart(){
+            if(this.quantity !== undefined && this.quantity !== 0){
+                this.quantity -= 1;
+            }
+            cart.removeItem(this);
+        }    
+
+        function addQuantityProperty(service) {
+            // default service quantity
+            service.quantity = 0;
+            service.addToCart = addToCart;
+            service.removeFromCart = removeFromCart;
+        }
+        // fn to add a property isAddedToCart in package
+        function addIsAddedToCartProperty(package){
+            package.isAddedToCart = false;
+            package.addToCart = addToCart;
+            package.removeFromCart = removeFromCart;
+        }
+        // if product is in cart update cart price on reload
+        function updateCartPrice(productinstance){
+            if(cart.hasItem(productinstance)){
```

Comment:
This is not the best way to check if the cart has productinstance.
` 
if( productinstance in cart && cart.productinstance)
` 
and yes the name should be camelcased productInstance